README.md

```diff
@@ -59,6 +59,32 @@ $ openssl genrsa 2048 > ../localhost.key
 $ openssl req -new -x509 -nodes -sha256 -days 3650 -key ../localhost.key -subj '/CN=*.localhost' > ../localhost.cert
 ```
 
+### Single-user server with Docker
+
+(First, install [Docker](https://docker.com)).
+
+Self-signed SSL cert and key are generated at build time (not for production! You can override this during the `run` step). The root ACL is also generated, with the owner being the WebID passed in:
+
+```bash
+$ sudo docker build --build-arg admin_user=https://yourwebsite.com/#me -t ldnode .
+```
+
+This `admin_user` needs to be a real WebID with the corresponding certificate installed in your browser.
+
+Then `run`, mounting the directory in which your data will be contained (this can be anywhere on your local machine):
+
+```bash
+$ sudo docker run -d -p 8443:8443 -v /path/to/data:/src/data --name my-ldnode ldnode
+```
+
+If you already have an SSL cert (eg. from LetsEncrypt), make sure they are named `ssl-cert.pem` and `ssl-key.pem` and mount the containing directory for those as well:
```

Comment:
I would suggest noting that the cert and key files should be *alone* in their containing directory.